docs/area-owners.md

```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
 | area-System.Data.OleDB                         | @ajcvickers   | @ajcvickers                                         |                                                                                                                                                                                                                                                                                           |
 | area-System.Data.SqlClient                     | @David-Engel  | @cheenamalhotra @david-engel @karinazhou @JRahnama  | Archived component - limited churn/contributions (see https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/)                                                                                                                                                  |
 | area-System.Diagnostics                        | @tommcdon     | @tommcdon @Anipik @ViktorHofer                      | Sub-Areas: <ul><li>System.Diagnostics.EventLog - @Anipik @ViktorHofer</li></ul>                                                                                                                                                                                                           |
+| area-System.Diagnostics-coreclr                | @tommcdon     | @tommcdon @Anipik @ViktorHofer                      | Sub-Areas: <ul><li>System.Diagnostics.EventLog - @Anipik @ViktorHofer</li></ul>                                                                                                                                                                                                           |
```

Comment:
It's unclear what these now represent, I'm not seeing anything left owned by my team.
``` suggestion
| area-System.Diagnostics                        | @tommcdon     | @tommcdon                       |                                                                                                                                                                                                            |
| area-System.Diagnostics-coreclr                | @tommcdon     | @tommcdon                       |                                                                                                                                                                                                            |
```